api/helpers/utils.js

```diff
@@ -89,8 +89,87 @@ const updateJuvenileName = (name, parentArrayName = []) => {
   return updatedName;
 };
 
+const nyCities = ["New York", "New York City", "NYC"];
+const nyCounties = ["Richmond", "Queens", "New York", "Kings", "Bronx"];
+const nyStates = ["NY", "New York"];
```

Comment:
Why do we need the options--are users allowed to enter the states in multiple formats, or...?

Reply:
True, I'll remove "New York" because that did give me issues.